airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -513,12 +518,25 @@ def streams(self, config: MutableMapping[str, Any]) -> List[Stream]:
                 },
                 **args,
             ),
-            StripeLazySubStream(
+            UpdatedCursorIncrementalStripeLazySubStream(
                 name="invoice_line_items",
                 path=lambda self, stream_slice, *args, **kwargs: f"invoices/{stream_slice['parent']['id']}/lines",
                 parent=invoices,
+                event_types=[
+                    "invoice.created",
+                    "invoice.deleted",
+                    "invoice.updated",
+                    # the event type = "invoice.upcoming" doesn't contain the `primary_key = `id` field,
+                    # thus isn't used, see the doc: https://docs.stripe.com/api/invoices/object#invoice_object-id
+                    # reference issue: https://github.com/airbytehq/oncall/issues/5560
+                ],
                 sub_items_attr="lines",
-                slice_data_retriever=lambda record, stream_slice: {"invoice_id": stream_slice["parent"]["id"], **record},
+                slice_data_retriever=lambda record, stream_slice: {
+                    "invoice_id": stream_slice["parent"]["id"],
+                    "created": stream_slice["parent"]["created"],
```

Comment:
@alafanechere can we validate that the slice retriever is useful here for datetime fields? It seems like we don't need it for bank account and the Stripe bank account object does not contain de `created` and we don't have that.

Also, can we rename these to clarify that those values are for invoices and not invoice_line_items?